lib/plugins/validation.js

```diff
@@ -178,66 +177,61 @@ var Validator = define(null, {
 
         isEmail:function isEmail(opts) {
             return this.__addAction(function (col) {
-                return validatorCheck(col).isEmail();
+                return validator.isEmail(col);
             }, merge({message:"{col} must be a valid Email Address got {val}"}, opts));
         },
 
         isUrl:function isUrl(opts) {
             return this.__addAction(function (col) {
-                return validatorCheck(col).isUrl();
+                return !!col.match(/^(?:https?|ftp):\/\/localhost/) || validator.isURL(col);
             }, merge({message:"{col} must be a valid url got {val}"}, opts));
```

Comment:
New valididator.isURL function excludes hostnames w/o tld be default.  The old function did too but allowed localhost, so I specifically allow  for it here.